project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -92,9 +92,9 @@ function c4m_user_notifications_group_status_set_softdeleted($node, $group_forme
 }
 
 /**
- * Email notification about organic group deletion.
+ * Email notification to admins about organic group deletion.
  */
```

Comment:
@anvmn 
can you fix the docs so it will document the arguments as well ?
thanks